extensions/ql-vscode/test/vscode-tests/cli-integration/run-cli.test.ts

```diff
@@ -42,6 +47,23 @@ describe("Use cli", () => {
     expect(result).toEqual(["-J-Xmx4096M", "--off-heap-ram=4096"]);
   });
 
+  describe("silent logging", () => {
+    it("should log command output", async () => {
```

Comment:
I could have made these use [`itWithCodeQL`](https://github.com/github/vscode-codeql/blob/256b806cd47b27b430d9ed4fe785cbe2d1fd7b14/extensions/ql-vscode/test/vscode-tests/cli.ts#L31) (which checks whether the CodeQL libraries are checked out) instead of `it`. But I don't actually care whether `resolve queries` finds anything, just whether it was run!